js/WKWebView.ios.js

```diff
@@ -15,14 +15,20 @@ import { requireNativeComponent } from 'react-native';
 
 const RCTWKWebView = requireNativeComponent('RCTWKWebView');
 
-class WKWebView extends React.Component {
-  componentWillReceiveProps(nextProps) {
+type RCTWKWebViewProps = {
+  allowsInlineMediaPlayback?: boolean,
```

Comment:
Something we've started doing more of (but are still not at 100% on) internally is making sure that optional props are also nullable:

```
  allowsInlineMediaPlayback?: ?boolean,
```

The reason for this is so that you can have a parent component which can spread in it's props:

```
const MyWrappedComponent(props: RCTWKWebViewProps) => (
   <WKWebView {...props} />
)
```

The above will error with the types you have right now because it might be passing the prop but setting it to null having come from the wrapper. If you make the webview props both optional and nullable then this will work fine.